misc/manifest.xml

```diff
@@ -2,7 +2,7 @@
 <ExtensionImage xmlns="http://schemas.microsoft.com/windowsazure">
   <ProviderNameSpace>Microsoft.GuestConfiguration</ProviderNameSpace>
   <Type>ConfigurationForLinux</Type>
-  <Version>1.24.1</Version>
+  <Version>1.25.2</Version>
```

Comment:
>2 [](start = 16, length = 1)

latest version is 1.25.5
and this also should not b changed if binaries (zip file is not changed)

Reply:
Updating PR. IDK why either of these files are appearing in the changed file list.

Reply:
Solved by running a rebase against Master